apps/portal/src/app/features/file-processor/file-processor.component.scss

```diff
@@ -1,5 +1,5 @@
 .table-container {
```

Comment:
Add title heading to the page not just browser title.
Add what files are accepted and max size of the file.

Reply:
Use different color of SUmmarize me(like Success)

Reply:
1. added title heading for page
2. used primeNg p-message to show info valid file types and max size allowed
3.used success for summarize file

(also did some changes after adding p-message to make UI consistent)